### ISSUE TITLE
Translation ist not working

### ISSUE THREAD

**Issue by Christianzimpel** [proteusthemes/one-click-demo-import#169]
Hi, 
I tranlated a Theme and other Plugins. Everything goes well with poe edit. 
But with this Plugin, it dos not take the translated text. 
Someone any idea?

**Comment by capuderg** [comment]
Hi,

I see the issue. You are trying to translate the text on the plugin page, but these texts don't exist in the pot file...

We'll fix that and update the plugin in a few days.

Take care!

**Comment by Christianzimpel** [comment]
It is not working.  did you try it to make a translation with poedit? 
Just one word to see if it work? Would be awesome. If I know it is working with you. I can looking on my server etc...

**Comment by capuderg** [comment]
Yes, I just tried it. Created a new "German" translation in the Poedit software (from the new .pot file). Named the files `pt-ocdi-de_DE.po` and `pt-ocdi-de_DE.mo` and uploaded them to the `.../wp-content/plugins/one-click-demo-import/languages/` folder. Changed the language in wp-admin dashboard settings to "Deutsch" and it worked.
